app.json

```diff
@@ -12,22 +12,18 @@
   "category": [
     "video"
   ],
-  "permissions": [
-    "homey:manager:speech-input",
-    "homey:manager:speech-output"
-  ],
   "images": {
     "large": "/assets/images/large.png",
     "small": "/assets/images/small.png"
   },
   "author": {
-    "name": "Ricardo Snoek",
-    "email": "me@ricardoismy.name"
+    "name": "Tapio Heiskanen",
+    "email": "oh2th@iki.fi"
   },
   "contributors": {
     "developers": [
       {
-        "name": "Ricardo Snoek",
+        "name": "Ricardo Snoek (legacy)",
         "email": "me@ricardoismy.name"
       },
       {
```

Comment:
I feel like I'm being shown the door 😄 

![Get out!](https://media.giphy.com/media/ac7MA7r5IMYda/giphy.gif)

Reply:
Yep, sorry for that. At the time I was with the impression that the app was abandoned. I'll fix this back in the sdk3 branch.

Reply:
Restored this